### ISSUE TITLE
HTTP/2 Support?

### ISSUE THREAD

**Issue by suchorski** [Kong/unirest-java#476]
There is a way to support HTTP 2?

**Comment by ryber** [comment]
The new release candidates support HTTP2, but require at least Java 11

**Comment by suchorski** [comment]
I'm using Java 17.

The RC13? In maven central has only RC2.

And how to use it?

Thanks!

**Comment by ryber** [comment]
The RC versions have been re-packaged to avoid conflicts. see https://github.com/Kong/unirest-java/blob/jv11/UPGRADE_GUIDE.md

**Comment by ryber** [comment]
https://mvnrepository.com/artifact/com.konghq/unirest-java-core

**Comment by suchorski** [comment]
Nice... will take a look

**Comment by suchorski** [comment]
Need to set something to use the HTTP 2?
How to check if is using it?
`Unirest.config().getVersion()` returns `null`

Thanks

**Comment by suchorski** [comment]
O figured out the endpoint I was accessing doesn't support TSLv1.3 and my SO updated and Java is using it. Using Postman I could disable the TSLv1.3 but can't do this with Unirest

**Comment by ryber** [comment]
You can in fact disable TLS ```Unirest.config().verifySsl(false)```

You don't need to set or do anything for HTTP2, it just does it as part of negotiation

**Comment by suchorski** [comment]
I think the problem is not with SSL. My code works fine in my windows environment. When I deploy to the server I got 403 from API that I'm accessing.
Using Postman on my Windows. I got 403 when I do a Post on /login with defaults, and 200 when I disable the TLS v1.3 protocol.

I tried Unirest.config().protocols("TLSv1.2") but I got 403 even on Windows and my server.

Did you see something like that to help me?

On the Postman woths with TLS 1.2 and Cipher ECDHE-ECDSA-AES128-GCM-SHA256
